packages/c/crun/package.yml

```diff
@@ -0,0 +1,27 @@
+name       : crun
+version    : 1.16.1
+release    : 1
+source     :
+    - https://github.com/containers/crun/releases/download/1.16.1/crun-1.16.1.tar.zst : 473968be42b35eaf9477a11855b1deaa3e1072b0604a5c20a470e2f108280afb
+homepage   : https://github.com/containers/crun
+license    : |
+    - GPL-2.0-only
+    - LGPL-2.1-only
+component  : virt
+summary    : A fast and low-memory footprint OCI Container Runtime fully written in C. 
+description: |
+    - A fast and lightweight fully featured OCI runtime and C library for running containers
+builddeps  :
+    - pkgconfig(libcap-ng)
+    - pkgconfig(libseccomp) 
+    - pkgconfig(libsystemd)
+    - pkgconfig(yajl) 
+    - go-md2man
+setup      : |
+    %autogen
+    %configure --enable-shared
+build      : |
+    %make
+install    : |
+    %make_install
+    rm $installdir/usr/lib64/libcrun.a
```

Comment:
Add `--disable-static` to the configure options and then remove this `rm` statement